ras_backstage/controllers/controller.py

```diff
@@ -8,9 +12,11 @@
 log = get_logger()
 
 
+JWT_ALGORITHM = 'HS256'
```

Comment:
On front stage the JWT_ALGORITHM was extracted and used within the jwt.py file to keek all that in the same place. How do you feel about placing a TODO on this so that we can do something similar with the backstage component?

Reply:
Sounds reasonable to me, I imagine we'd just need to choose an env var name that we use in all services requiring it.